data/comid/README.md

```diff
@@ -0,0 +1,83 @@
+
+# CoMID Template Format
+
+## 1 Introduction
+
+**CoMID (Concise Model Identifier)**,  is a data model and serialization format (in JSON) for capturing **reference values** and **verification keys** that can be used in remote attestation and other trust-verification scenarios. By standardizing how measurements are captured and shared, CoMID facilitates **interoperability**, **integrity**, and **traceability** across various systems and vendors.
+
+## 2 Template Structure
+
+A CoMID template is a JSON document composed of **top-level fields** and a **triples** object. The **top-level fields** provide overall identification, language, and authorship, while the **triples** object contains domain-specific data (e.g., reference values, attester keys).
+
+```
+{
+  "lang": "<language-region>",
+  "tag-identity": { ... },
+  "entities": [ ... ],
+  "triples": {
+    "reference-values": [ ... ],
+    "attester-verification-keys": [ ... ]
+    ...
+  }
+}
+``` 
+
+### 2.1 Top-Level Fields
+
+-   **lang** (`String`): Defines the language or locale (e.g., `"en-GB"`).
+-   **tag-identity** (`Object`): Uniquely identifies this CoMID document via an ID (often a UUID) and includes a version number.
```

Comment:
Tag identifiers don’t have versions according to the spec.

Reply:
as in template, tag-identify have version information in it 
```
"tag-identity": {
    "id": "43BBE37F-2E61-4B33-AED3-53CFF1428B16",
    "version": 0
  },
```